pandas/tests/io/formats/test_format.py

```diff
@@ -194,6 +194,11 @@ def check(null_counts, result):
             check(True, False)
             check(False, False)
 
+        with tm.assert_produces_warning(FutureWarning):  # GH37999
```

Comment:
``tm.assert_produces_warning`` now has an optional ``match`` kwarg to match the warning message.
Could use it here to ensure that the proper warning message is issues.

Reply:
Added